dev/com.ibm.ws.jpa.tests.jpa_32_fat/test-applications/jakartadata/src/io/openliberty/jpa/data/tests/web/JakartaDataRecreateServlet.java

```diff
@@ -370,6 +371,78 @@ public void testOLGH28931() throws Exception {
         assertEquals("IBM Rochester", result.name);
         assertEquals(55901, result.location.address.zip);
     }
+    @Test
+    public void testEclipseLinkSnapshot20240808() throws Exception {
```

Comment:
This issue was raised when we tried to integrate the previous version of Snapshot August 8 version. Should we remove this test ?

Reply:
You should keep the test, but rename it to match the number of the issue that was raised, or the pr that fixed it. All the tests in this class should be mapped to some documented issue so someone can see how it failed and how it was fixed.

Reply:
For example, testOLGH29117, is for open-liberty git hub issue 29117. so, if this can be mapped to an open liberty issue, then use OLGHxxxxx, or if it is just an EclipseLink issue, then use ELGHxxxxx.